uiplib/scheduler.py

```diff
@@ -17,7 +17,7 @@
 from threading import Thread
 
 
-class scheduler:
+class Scheduler:
```

Comment:
This isn't the only change. You've to change all other instances where this class is being used.